app/models/chargeback_rate.rb

```diff
@@ -70,4 +72,9 @@ def self.seed
       end
     end
   end
+
```

Comment:
@GregP And we probably have to unwind each one separately, each in a separate PR.  Over time, my vision is to have the UI go through the REST API exclusively (like the new self-service UI that we just added).